frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -257,89 +257,76 @@ describe("scenarios > dashboard > dashboard drill", () => {
       .within(() => cy.findByText("foo"));
   });
 
-  it("should pass multiple filters for numeric column on drill-through (metabase#13062)", () => {
-    // Preparation for the test: "Arrange and Act phase" - see repro steps in #13062
-    // 1. set "Rating" Field type to: "Category"
+  ["field value", "card title"].forEach(testCase => {
+    it(`should pass multiple filters for numeric column on drill-through the ${testCase} (metabase#13062)`, () => {
+      // Set "Rating" Field type to: "Category"
+      cy.request("PUT", `/api/field/${REVIEWS.RATING}`, {
+        semantic_type: "type/Category",
+      });
 
-    cy.request("PUT", `/api/field/${REVIEWS.RATING}`, {
-      semantic_type: "type/Category",
-    });
-    // 2. create a question based on Reviews
-    cy.request("POST", `/api/card`, {
-      name: "13062Q",
-      dataset_query: {
-        database: 1,
+      const questionDetails = {
+        name: "13062Q",
         query: {
           "source-table": REVIEWS_ID,
         },
-        type: "query",
-      },
-      display: "table",
-      visualization_settings: {},
-    }).then(({ body: { id: questionId } }) => {
-      cy.createDashboard("13062D").then(({ body: { id: dashboardId } }) => {
-        // add filter to the dashboard
-        cy.request("PUT", `/api/dashboard/${dashboardId}`, {
-          parameters: [
-            {
-              id: "18024e69",
-              name: "Category",
-              slug: "category",
-              type: "category",
-            },
-          ],
-        });
+      };
+
+      const filter = {
+        id: "18024e69",
+        name: "Category",
+        slug: "category",
+        type: "category",
+      };
+
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: { id, card_id, dashboard_id } }) => {
+          // Add filter to the dashboard
+          cy.request("PUT", `/api/dashboard/${dashboard_id}`, {
+            parameters: [filter],
+          });
 
-        // add previously created question to the dashboard
-        cy.request("POST", `/api/dashboard/${dashboardId}/cards`, {
-          cardId: questionId,
-        }).then(({ body: { id: dashCardId } }) => {
-          // connect filter to that question
-          cy.request("PUT", `/api/dashboard/${dashboardId}/cards`, {
+          // Connect filter to the dashboard card
+          cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
```

Comment:
Looks like even this could be done in the `describe` block, before the `it`s, no?

Reply:
Yes, it goes to the `beforeEach()` hook.